tests/platform_tests/test_intf_fec.py

```diff
@@ -19,6 +19,15 @@
 ]
 
 
+def is_supported_platform(duthost):
```

Comment:
I would define this as a fixture, and then it should not be called explicitly from the test

Reply:
Done. Thanks!